common/metrics/config.go

```diff
@@ -202,17 +202,59 @@ var (
 		200000 * ms,
 		500000 * ms,
 		1000000 * ms,
-		2000000 * ms,
-		5000000 * ms,
-		10000000 * ms,
```

Comment:
I assume this is recent change after 1.16?

Reply:
Yes